app/views/facilities/_form.html.erb

```diff
@@ -7,7 +7,7 @@
             <div class="grid grid-cols-6 gap-6">
               <div class="col-span-6 sm:col-span-3">
                 <%= form.label :kind, "Type", class: "block text-sm font-medium text-gray-700" %>
-                <%= form.select("kind", ["PHC", "CHC", "Hospital"] , {}, onChange: "removeCHCSelect(event)", required: true, class: "mt-1 block w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm")%>
+                <%= form.select("kind", [Facility.kinds[:primary], Facility.kinds[:secondary], Facility.kinds[:hospital]] , {}, onChange: "removeCHCSelect(event)", required: true, class: "mt-1 block w-full py-2 px-3 border border-gray-300 bg-white rounded-md shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm")%>
```

Comment:
`Facility.kinds.values` will return the same list

Reply:
done